frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -371,14 +387,26 @@ pub mod pallet {
         /// This must be called by the developer who registered the contract.
         ///
         /// Warning: After this action contract can not be assigned again.
-        #[pallet::weight(T::WeightInfo::unregister(T::MaxNumberOfStakersPerContract::get()))]
+        #[pallet::weight(T::WeightInfo::unregister())]
         pub fn unregister(
```

Comment:
once we get rid of Sudo, how can an admin do unregister for contracts that prove to not be worked on or malicious?

Reply:
Sure, I will change it to root-only.
We will also need a couple for root-only force methods in addition to this.